### PR TITLE
Feat-78 jwt 인증 및 인가 실패 예외 처리 방법 수정

### PR DESCRIPTION
**Pull request by EunjungOh123**
CustomAccessDeniedHandler와 CustomAuthenticationEntryPoint에서 바로 에러 응답 내려주는 방식으로 변경

### CODE REVIEW

File: src/main/java/com/gaethering/gaetheringserver/domain/member/jwt/auth/CustomAccessDeniedHandler.java [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
         log.info("UnAuthorized!!! " + accessDeniedException.getMessage());
-        response.sendRedirect("/exception/accessDenied");
+        ErrorResponse errorResponse
+                = new ErrorResponse(FAIL_TO_AUTHORIZATION.getCode(),
+                FAIL_TO_AUTHORIZATION.getMessage());
+
+        response.setContentType("application/json;charset=UTF-8");
+        response.setStatus(HttpStatus.UNAUTHORIZED.value());
+
+        PrintWriter writer = response.getWriter();
```
Comment (naekang) [inline src/main/java/com/gaethering/gaetheringserver/domain/member/jwt/auth/CustomAccessDeniedHandler.java:39]: log.error로 남기는 방식은 어떤가요? 지금 방식이랑 System.out.println이랑 비슷한거같아서 뭔가 다른 방법이 필요해보입니다! 😄

Comment (EunjungOh123) [reply src/main/java/com/gaethering/gaetheringserver/domain/member/jwt/auth/CustomAccessDeniedHandler.java:39]: 에러 로그는 위에서 log.info로 남기고 있고 해당 코드는 json으로 response body 에러 응답을 내려주기 위한 설정입니다!

**Comment by naekang** [comment]
|File|Coverage [39.02%]|:x:|
|:-|:-:|:-:|
|[JwtAuthenticationFilter.java](https://github.com/Gaethering/Gaethering-Server/blob/698429810f0d8d264a40cc249f0b9eb33ce36db9/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2FJwtAuthenticationFilter.java)|68.29%|:x:|
|[CustomAccessDeniedHandler.java](https://github.com/Gaethering/Gaethering-Server/blob/698429810f0d8d264a40cc249f0b9eb33ce36db9/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2Fauth%2FCustomAccessDeniedHandler.java)|24.39%|:x:|
|[CustomAuthenticationEntryPoint.java](https://github.com/Gaethering/Gaethering-Server/blob/698429810f0d8d264a40cc249f0b9eb33ce36db9/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2Fauth%2FCustomAuthenticationEntryPoint.java)|24.39%|:x:|

|Total Project Coverage|67.88%|:x:|
|:-|:-:|:-:|

**Comment by naekang** [comment]
|File|Coverage [70.57%]|:x:|
|:-|:-:|:-:|
|[MemberServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/8d3a232f5c552de86d2907f784e3aed616fdb507/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fservice%2Fmember%2FMemberServiceImpl.java)|89.05%|:green_apple:|
|[JwtAuthenticationFilter.java](https://github.com/Gaethering/Gaethering-Server/blob/8d3a232f5c552de86d2907f784e3aed616fdb507/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2FJwtAuthenticationFilter.java)|68.29%|:x:|
|[CustomAccessDeniedHandler.java](https://github.com/Gaethering/Gaethering-Server/blob/8d3a232f5c552de86d2907f784e3aed616fdb507/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2Fauth%2FCustomAccessDeniedHandler.java)|24.39%|:x:|
|[CustomAuthenticationEntryPoint.java](https://github.com/Gaethering/Gaethering-Server/blob/8d3a232f5c552de86d2907f784e3aed616fdb507/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fjwt%2Fauth%2FCustomAuthenticationEntryPoint.java)|24.39%|:x:|

|Total Project Coverage|67.93%|:x:|
|:-|:-:|:-:|

**Pull request merged**